pinot-core/src/main/java/org/apache/pinot/core/operator/transform/TransformOperator.java

```diff
@@ -51,11 +54,13 @@
    * @param expressions Set of expressions to evaluate
    */
   public TransformOperator(@Nonnull ProjectionOperator projectionOperator,
-      @Nonnull Set<TransformExpressionTree> expressions) {
+      @Nonnull List<TransformExpressionTree> expressions) {
```

Comment:
Does the existing implementation break due to use of Set (ie when transforms are duplicated)? If not, what's the reason to change to List?

Reply:
Yes, I remember that maintaining order was important. Can't remember why. I might have fixed that. Will look at the code again.

Reply:
Its not important in terms of functionality but it matches the order in which we display the results for selection query(maintains the order in which columns appear in the query)